lens/lotusrepo/repo.go

```diff
@@ -3,23 +3,34 @@ package lotusrepo
 import (
 	"context"
 	"fmt"
+	"io"
 
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-fil-markets/storagemarket"
+	"github.com/filecoin-project/go-multistore"
 	"github.com/filecoin-project/go-state-types/abi"
 	"github.com/filecoin-project/sentinel-visor/lens"
+	peer "github.com/libp2p/go-libp2p-peer"
 	"github.com/urfave/cli/v2"
 
 	"github.com/filecoin-project/lotus/api"
+	"github.com/filecoin-project/lotus/chain/store"
 	"github.com/filecoin-project/lotus/chain/vm"
+	"github.com/filecoin-project/lotus/extern/sector-storage/ffiwrapper"
 	"github.com/filecoin-project/lotus/lib/cachebs"
-	"github.com/filecoin-project/lotus/node"
+	"github.com/filecoin-project/lotus/lib/ulimit"
+	marketevents "github.com/filecoin-project/lotus/markets/loggers"
 	"github.com/filecoin-project/lotus/node/impl"
 	"github.com/filecoin-project/lotus/node/repo"
+	"github.com/filecoin-project/specs-actors/actors/runtime/proof"
 	"github.com/filecoin-project/specs-actors/actors/util/adt"
+	"github.com/ipfs/go-cid"
+	blockstore "github.com/ipfs/go-ipfs-blockstore"
```

Comment:
Wrong blockstore:
https://godoc.org/github.com/filecoin-project/lotus/lib/blockstore

Reply:
the definition an alias type though, right?

Reply:
I didn't dig hat you pulled: you might have just grabbed aliases and it is fine. But for future-self...